code-challenges.test.js

```diff
@@ -10,45 +12,102 @@
 // Reminder: The test will call your function
 // Run the file with the following command: $ yarn jest
 
-
 // --------------------1) Create a function that takes in an array, removes the first item from the array and shuffles the remaining content.
 
 // a) Create a test with an expect statement using the variable provided. HINT: Check out this resource: https://jestjs.io/docs/expect#expectarraycontainingarray
-
-var colors1 = ["purple", "blue", "green", "yellow", "pink"]
-// Expected output example (can be a different order): ["yellow", "blue", "pink", "green"]
-var colors2 = ["chartreuse", "indigo", "periwinkle", "ochre", "aquamarine", "saffron"]
-// Expected output example (can be a different order): ["saffron", "aquamarine", "periwinkle", "indigo", "ochre"]
-
-
+describe("removeFirstItemAndShuffle", () => {
+  var colors1 = ["purple", "blue", "green", "yellow", "pink"];
+  // Expected output example (can be a different order): ["yellow", "blue", "pink", "green"]
+  var colors2 = [
+    "chartreuse",
+    "indigo",
+    "periwinkle",
+    "ochre",
+    "aquamarine",
+    "saffron",
+  ];
+  // Expected output example (can be a different order): ["saffron", "aquamarine", "periwinkle", "indigo", "ochre"]
+  it("takes in an array, removes the first item from the array and shuffles the remaining content", () => {
+    expect(removeFirstItemAndShuffle(colors1)).toEqual(
+      expect.arrayContaining(["yellow", "blue", "pink", "green"])
+    );
+    expect(removeFirstItemAndShuffle(colors2)).toEqual(
+      expect.arrayContaining([
+        "saffron",
+        "aquamarine",
+        "periwinkle",
+        "indigo",
+        "ochre",
+      ])
+    );
+  });
+});
 
 // b) Create the function that makes the test pass.
 
+// create function that takes in an array as the argument (done)
+// remove the first item from array with .shift() (done)
+// use a custom .sort() with Math.random to shuffle it up and return (done)
+// NOTE: after doing somer research, I'm reaching that this method isn't THE most ideal, something called the Fisher-Yates algorithm gives a better randomization, but I figure this is probably fine for the scope of this question...
 
+const removeFirstItemAndShuffle = (array) => {
+  array.shift();
+  return array.sort(() => Math.random() - 0.5);
+};
```

Comment:
I like this code snippet. Yes, it isn't the *most* random thing you can do but it is so concise.